### ISSUE TITLE
datacarpentry/python-socialsci

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#47]
tracking issues for <https://github.com/datacarpentry/python-socialsci>/

 - [x] link issues in episode 11 (see [below](#links))
 - [x] missing links in instructor notes (I think this is purposeful)
 - [x] missing file (guide/index.html)

## links

```markdown
161:episodes/05-processing-data-from-file.md:47 [uninformative link text]: [here](data/SAFI_results.csv)
162:episodes/05-processing-data-from-file.md:47 [uninformative link text]: [here](https://datacarpentry.org/socialsci-workshop/data/index.html)
163:episodes/08-Pandas.md:66 [uninformative link text]: [here](data/SN7577.tab)
164:episodes/08-Pandas.md:66 [uninformative link text]: [here](https://beta.ukdataservice.ac.uk/datacatalogue/studies/study?id=7577&type=data%20catalogue#!/details)
165:episodes/11-joins.md:128 [missing file]: [SQL lesson](./episodes/sql...)
166:instructors/instructor-notes.md:32 [missing file]: [Introduction to Python](link)
167:instructors/instructor-notes.md:42 [missing file]: [Python basics ](link)
168:instructors/instructor-notes.md:56 [missing file]: [Python control structures](link)
169:instructors/instructor-notes.md:62 [missing file]: [Creating re-usable code](link)
170:instructors/instructor-notes.md:72 [missing file]: [Processing data from a file](link)
171:instructors/instructor-notes.md:93 [missing file]: [Date and Time](link)
172:instructors/instructor-notes.md:103 [missing file]: [Processing JSON data](link)
173:instructors/instructor-notes.md:121 [missing file]: [Pandas](link)
174:instructors/instructor-notes.md:131 [missing file]: [Extracting rows and columns](link)
175:instructors/instructor-notes.md:137 [missing file]: [Aggregations_and_missing_data](link)
176:instructors/instructor-notes.md:151 [missing file]: [Joins](link)
177:instructors/instructor-notes.md:165 [missing file]: [Long_and_wide_data_formats](link)
178:instructors/instructor-notes.md:175 [missing file]: [Data visualisation using Matplotlib](link)
179:instructors/instructor-notes.md:189 [missing file]: [Accessing SQLite Databases](link)

```

**Comment by zkamvar** [comment]
There is nothing to be done for this one... it's in alpha and a bunch of links will break.

**Comment by zkamvar** [comment]
fixed in b24146086b28c37de19aa1213a79e604eeedcf71
